### DISCUSSION TITLE
Honeywell vista support?

### DISCUSSION THREAD

**Discussion by adamurban** [taligentx/dscKeybusInterface#156] [dscKeybusInterface Discussion]
Commercial products like AlarmDecoder seem to be able to support both DSC and Honewell/Ademco protocols via the same physical interface. Perhaps this project could too?

So far these are the only details I could find on the honeywell protocol: https://forum.arduino.cc/index.php?topic=200820.0
Looks like it is RS232, 4800, 8 data bits, Even parity, 2 stop bits...

**Comment by Dilbert66** [comment]
here you go:

https://github.com/Dilbert66/esphome-vistaECP

They are completely different systems.   No need to try and combine software when you can just pick the version you want and use that.    The DSC bus is quite a bit more complex  than the vista ECP bus and the hardware is slightly different too for interfacing with the lines.   Yes, if I was paying over $100 dollars for a piece of hardware, i'd want to it to do everything  but in this case, there is no need.   It just adds more uneeded complexity to the software.

**Comment by taligentx** [comment]
I've added a link to @Dilbert66's esphome-vistaECP project in the README.md, support for other security systems is currently out of scope for this library - if nothing else, I don't have the hardware to test.  If someone comes along with a PR that seamlessly integrates the protocol decoding for a different system (for example, something that doesn't inflate the compilation size for DSC systems or require changes in the existing example sketches), I'm open to merging.
